docs/contributing/docs-and-website-components.md

```diff
@@ -0,0 +1,409 @@
+      title: "Docs and Website Components",
+      tableOfContentsDepth: 2,
+    },
+    tableOfContents: {
+      items: [
+        {
+          url: "#globally-available-components",
+          title: "Globally available components",
+          items: [
+            {
+              url: "#guide-list",
+              title: "Guide list",
+            },
+            {
+              url: "#egghead-embed",
+              title: "Egghead embed",
+            },
+            {
+              url: "#pull-quote",
+              title: "Pull quote",
+            },
+          ],
+        },
+        {
+          url: "#other-available-components",
+          title: "Other available components",
+          items: [
+            {
+              url: "#layer-model",
+              title: "Layer model",
+            },
+            {
+              url: "#horizontal-navigation-list",
+              title: "Horizontal navigation list",
+            },
+            {
+              url: "#breadcrumb",
+              title: "Breadcrumb",
+            },
+            {
+              url: "#table-of-contents",
+              title: "Table of Contents",
+            },
+          ],
+        },
+        {
+          url: "#writing-content-in-markdown",
+          title: "Writing content in markdown",
+          items: [
+            {
+              url: "#code-blocks",
+              title: "Code blocks",
+            },
+          ],
+        },
+        {
+          url: "#styling-components-on-gatsbyjsorg-with-theme-ui",
+          title: "Styling components on Gatsbyjs.org with Theme-UI",
+          items: [
+            {
+              url: "#design-tokens",
+              title: "Design tokens",
+            },
+            {
+              url: "#the-sx-prop",
+              title: "The sx prop",
+            },
+          ],
+        },
+      ],
+    },
+  }}
+/>
+
+---
+
+## Writing content in Markdown
+
+New docs and blog posts are added to the [docs](https://github.com/gatsbyjs/gatsby/tree/master/docs/) folder inside the Gatsby repository. They are stored as `.md` (Markdown) or `.mdx` (MDX) files and can be written using Markdown, or using inline JSX thanks to MDX. Writing in Markdown will output tags that are styled according to [Gatsby's design guidelines](/guidelines/color/).
+
+You can read more about writing in Markdown in the [Markdown syntax guide](/docs/mdx/markdown-syntax).
+
+### Code blocks
+
+Code can be formatted using regular Markdown syntax, but the docs site has additional enhancements that can be used thanks to various Gatsby plugins that aren't all native Markdown.
+
+#### Usage
+
+Code blocks are wrapped in 3 backticks. A language can be added immediately after the first set of back ticks to provide syntax highlighting for the language. A title of the file can be added after the language. Line highlighting can be included in the code block by commenting `highlight-line`, or `highlight-start` followed by a `highlight-end`.
```

Comment:
```suggestion
Code blocks are wrapped in 3 backticks. A language can be added immediately after the first set of back ticks to provide syntax highlighting for the language. A `title` of the file can be added after the language. Line highlighting can be included in the code block by commenting `highlight-line`, or `highlight-start` followed by a `highlight-end`.
```
Could we add common languages so people know what the conventions are (`sh`, `javascript`, `markdown`, etc.)?